testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1525,13 +1530,13 @@
 
 When(/^I execute "([^"]*)" on the "([^"]*)"$/) do |command, host|
   node = get_target(host)
-  command_output, _code = node.run(command, check_errors: true, verbose: true)
+  node.run(command, check_errors: true, verbose: true)
 end
 
 When(/^I check the cloud-init status on "([^"]*)"$/) do |host|
   node = get_target(host)
-  _hostname, local, remote, node_code = node.test_and_store_results_together('hostname', 'root', 500)
-  command_output, _code = node.run("cloud-init status --wait", check_errors: true, verbose: false)
+  node.test_and_store_results_together('hostname', 'root', 500)
+  node.run("cloud-init status --wait", check_errors: true, verbose: false)
```

Comment:
Any reason, why you remove the variables that store the return values of these methods?

Reply:
They were not used after been assigned.